src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -700,100 +753,119 @@
+  "fluid.tconstruct.molten_silver.fluid_effect": "熔融银可以排出毒素而不波及正面效果。",
 
   "block.tconstruct.molten_nickel_fluid": "熔融镍",
   "fluid.tconstruct.molten_nickel": "熔融镍",
   "fluid.tconstruct.flowing_molten_nickel": "流动的熔融镍",
   "item.tconstruct.molten_nickel_bucket": "熔融镍桶",
+  "fluid.tconstruct.molten_nickel.fluid_effect": "熔融镍可以攻击敌人并降低反击风险，这种流体有什么理由不爱呢？",
 
   "block.tconstruct.molten_zinc_fluid": "熔融锌",
   "fluid.tconstruct.molten_zinc": "熔融锌",
   "fluid.tconstruct.flowing_molten_zinc": "流动的熔融锌",
   "item.tconstruct.molten_zinc_bucket": "熔融锌桶",
+  "fluid.tconstruct.molten_zinc.fluid_effect": "熔融锌可能是提升速度的最佳选择，你的速度让对手望尘莫及。",
 
   "block.tconstruct.molten_platinum_fluid": "熔融铂",
   "fluid.tconstruct.molten_platinum": "熔融铂",
   "fluid.tconstruct.flowing_molten_platinum": "流动的熔融铂",
   "item.tconstruct.molten_platinum_bucket": "熔融铂桶",
+  "fluid.tconstruct.molten_platinum.fluid_effect": "熔融铂对敌人的挖掘速度削减作用尚不明晰，但其他功能仍待发掘。",
 
   "block.tconstruct.molten_tungsten_fluid": "熔融钨",
   "fluid.tconstruct.molten_tungsten": "熔融钨",
   "fluid.tconstruct.flowing_molten_tungsten": "流动的熔融钨",
   "item.tconstruct.molten_tungsten_bucket": "熔融钨桶",
+  "fluid.tconstruct.molten_tungsten.fluid_effect": "熔融钨是重金属之一，希望它能让敌人定在那里。",
 
   "block.tconstruct.molten_osmium_fluid": "熔融锇",
   "fluid.tconstruct.molten_osmium": "熔融锇",
   "fluid.tconstruct.flowing_molten_osmium": "流动的熔融锇",
   "item.tconstruct.molten_osmium_bucket": "熔融锇桶",
+  "fluid.tconstruct.molten_osmium.fluid_effect": "熔融锇可以定住敌人，任你处置。",
 
   "block.tconstruct.molten_uranium_fluid": "熔融铀",
   "fluid.tconstruct.molten_uranium": "熔融铀",
   "fluid.tconstruct.flowing_molten_uranium": "流动的熔融铀",
   "item.tconstruct.molten_uranium_bucket": "熔融铀桶",
+  "fluid.tconstruct.molten_uranium.fluid_effect": "熔融铀产生的放射性衰变可不安全，所以把它喷在你的敌人身上。",
 
   "_comment": "Compat Alloys",
 
   "block.tconstruct.molten_bronze_fluid": "熔融青铜",
   "fluid.tconstruct.molten_bronze": "熔融青铜",
   "fluid.tconstruct.flowing_molten_bronze": "流动的熔融青铜",
   "item.tconstruct.molten_bronze_bucket": "熔融青铜桶",
+  "fluid.tconstruct.molten_bronze.fluid_effect": "当你躲避幻翼的追踪时，熔融青铜便是最佳的深棕色饮料。",
 
   "block.tconstruct.molten_brass_fluid": "熔融黄铜",
   "fluid.tconstruct.molten_brass": "熔融黄铜",
   "fluid.tconstruct.flowing_molten_brass": "流动的熔融黄铜",
   "item.tconstruct.molten_brass_bucket": "熔融黄铜桶",
+  "fluid.tconstruct.molten_brass.fluid_effect": "熔融黄铜能帮你抵御高温；如果时机得当，喝第二口时并不会消耗。",
 
   "block.tconstruct.molten_electrum_fluid": "熔融琥珀金",
   "fluid.tconstruct.molten_electrum": "熔融琥珀金",
   "fluid.tconstruct.flowing_molten_electrum": "流动的熔融琥珀金",
   "item.tconstruct.molten_electrum_bucket": "熔融琥珀金桶",
+  "fluid.tconstruct.molten_electrum.fluid_effect": "熔融琥珀金在你预算紧张时绝对是高速采掘的不二之选。",
 
   "block.tconstruct.molten_invar_fluid": "熔融殷钢",
   "fluid.tconstruct.molten_invar": "熔融殷钢",
   "fluid.tconstruct.flowing_molten_invar": "流动的熔融殷钢",
   "item.tconstruct.molten_invar_bucket": "熔融殷钢桶",
+  "fluid.tconstruct.molten_invar.fluid_effect": "如果你想要吃掉那块强大蛋糕，那熔融殷钢或许有点作用。但总之，他是一个很奇怪的流体。",
```

```suggestion
  "fluid.tconstruct.molten_invar.fluid_effect": "如果你想要吃掉那块大蛋糕，那熔融殷钢或许有点作用。但总之，它是一个很奇怪的流体。",
```
Not quite sure what does "powerful" mean here, the suggestion take it as "of huge size, hard to finish(eat)" since I could only find it inflicting Hunger after searching the code, may require further testing.
The second change is required.

---

refered to the Slime cake in TiC3

---

```suggestion
  "fluid.tconstruct.molten_invar.fluid_effect": "如果你想要吃掉那块强力蛋糕，那熔融殷钢或许有点作用。但总之，它是一个很奇怪的流体。",
```